lib/elements/prompt.js

```diff
@@ -19,7 +19,7 @@ class Prompt extends EventEmitter {
     this.in = opts.stdin || process.stdin;
     this.out = opts.stdout || process.stdout;
     this.onRender = (opts.onRender || (() => void 0)).bind(this);
-    const rl = readline.createInterface(this.in);
+    const rl = readline.createInterface({ input:this.in, escapeCodeTimeout:50 });
```

Comment:
Because of this: https://github.com/nodejs/node/blob/master/lib/readline.js#L92

The default time out is too slow, and the user experience is like there are something wrong with the `prompt` performance.